bin/crystal

```diff
@@ -133,7 +110,7 @@ __warning_msg() {
 }
 
 
-SCRIPT_PATH="$(realpath "$0")"
+SCRIPT_PATH="$(readlink --canonicalize "$0")"
```

Comment:
No present on Alpine Linux, use `f` instead.

```
Usage: readlink [-fnv] FILE

Display the value of a symlink

        -f      Canonicalize by following all symlinks
        -n      Don't add newline
        -v      Verbose
```

Reply:
It's not Alpine: only `-f` is supported on OpenBSD and FreeBSD, for example, and seems to be somewhat portable (no guarantees, thought).

Thought `-f` doesn't seem to be supported on macOS. In fact I'm not sure `--canonicalize` is available until you install GNU coreutils...

I.e. the current script is portable, that's why it's here in the first place.